aptos-move/aptos-debugger/src/aptos_debugger.rs

```diff
@@ -99,6 +100,12 @@ impl AptosDebugger {
             |gas_meter| {
                 let gas_profiler = match txn.payload() {
                     TransactionPayload::Script(_) => GasProfiler::new_script(gas_meter),
+                    TransactionPayload::Automation(auto_payload) => GasProfiler::new_function(
+                        gas_meter,
```

Comment:
Is this `Unmertered` type? Or does this `gas_meter` actually meter positive gas?

Reply:
Yes it will meter positive gas utilizing production gas-meter
- https://github.com/Entropy-Foundation/aptos-core/blob/87dab4b4dfa179d188868881eb229ce7be6a10c1/aptos-move/aptos-vm/src/aptos_vm.rs#L1949
- https://github.com/Entropy-Foundation/aptos-core/blob/87dab4b4dfa179d188868881eb229ce7be6a10c1/aptos-move/aptos-vm/src/gas.rs#L107